### DISCUSSION TITLE
job.output.loglevel configuration

### DISCUSSION THREAD

**Discussion by marioaag** [avocado-framework/avocado#5152] [Q&A] [answered]
I'm modifying the job log level using the ~/.config/avocado/avocado.conf file:
```
[job.output]
loglevel = INFO
``` 

Following the documentation here:
https://avocado-framework.readthedocs.io/en/latest/config/index.html#config-reference

Without results, I sill see the debug logs in test results and the configuration as debug in the job log
```
2021-11-21 12:50:01,912 job              L0265 I    |  'job.output.loglevel': 'DEBUG',
```

Is anything that I'm missing?

**Comment by beraldoleal** [comment, marked as answer]
Hi @marioaag probably your config file is not being parsed properly. I just did a test and it is working as expecting.

To see if the file is being parsed, just run `avocado config | head`, the first lines shows you all the config files parsed.

You can force this  with the  `--config` option:

`$ avocado --config ~/.config/avocado/avocado.conf run /bin/true`

**Comment by marioaag** [comment]
Thanks for the hint! not sure what changed but it is working now!

**Comment by marioaag** [comment]
As a complement to this question...
I keep doing some validation and found that if this param is modified in the config file, it gets used by avocado starting from the second execution of avocado after it was modified.
As a workaround I'm passing the param directly to the JOB_CONFIG due I'm working directly to the with the API
